### ISSUE TITLE
Final processed recording cuts short by a minute or two (Strange)

### ISSUE THREAD

**Issue by stu1795** [ShareX/ShareX#2082]
I don't know why this is happening. I searched this on issues page but find none as a solution for this problem. 

What's happening --
1. Screen recording continues for lets say 10 minutes before being stopped.

2. Final processed recording is 9 minutes only. When the timer to record the screen reached 10 in front of me when I pressed the stop button. I tried this various times. Initially the final recordings came out to be only 8:07 & 8:36 minutes but then after I chose a custom capture screen smaller than the previous one, set fps to 60 and -rtbufsize to 1000M the final recording increased to 9 minutes.

I'm using Windows 7 and my ShareX setting is given below : -

![2016-11-13_04-26-11](https://cloud.githubusercontent.com/assets/23425072/20242596/ee23887a-a959-11e6-81c6-b277df450039.png)

Can someone help me with this issue ?

**Comment by stu1795** [comment]
Someone please help me with this. I am stuck in the middle of something really important.

**Comment by stu1795** [comment]
No need to reply. Found a solution to this. After doing extensive research and various experiment on this issue, finally I found a solution to this. I found out the total number of seconds which you lose for a clip after pressing stop. The number of seconds varied between 4-10 seconds for every 1 minute clip. Now, the safest number which I got after various experiments was 7 seconds. So, for every 1 minute clip keep 7 seconds buffer. 

For example,  let's say your total recording time at which point you want to stop the clip is 00:09:26. Then add 7 seconds multiplied by 10(total length of your clip), i.e. 70 seconds to your clip. 
What I mean by adding is let the recording continue for next 70 seconds more,i.e. stop the recording at 00:10:36. 
And, the final clip will contain more or less 00:09:26 only. 

So, just let the recording continue for 7 seconds per minute rate and your final clip will never be less than what you supposed to record. 

Enjoy all.
